### DISCUSSION TITLE
Coordinator 분리하는데 이름 추천 받습니다.

### DISCUSSION THREAD

**Discussion by never-better** [boostcampwm-2022/iOS09-burstcamp#183] [Ideas]
TabBarCoordinator 가 너무 커서 ScrapCoordinator와 MyPageCoordinator로 분리하려고 합니다.

큰 이견 없으면 ScrapCoordinator, MyPageCoordinator로 가겠습니다.

많은 의견 부탁드리겠습니다.

**Comment by guesswb** [comment]
제일 적합한 것 같습니다
